tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +50,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="num_workers", default=NUM_WORKERS, type="int",
```

Comment:
Good catch! I don't know how this possibly worked, but it did when I tested it. Will fix.